Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Materials/ShaderBallTriplanar.mat

```diff
@@ -6,142 +6,99 @@ Material:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
-  m_Name: ShaderBallGlass
+  m_Name: ShaderBallTriplanar
```

Comment:
Did you mean to remove the glass material?

Reply:
I did remove it and replaced it with the triplanar one since I thought the glass one was a little "meh" looking. But, I'm more than happy to add it back if you think it is useful?

Reply:
the nice thing about having the glass material is it shows customers (like me :) ) who may need an example, how to achieve a glass effect,

Reply:
Glass material is still there, correct? Just replaced with a better one?

Reply:
That's a very good point! I'll add it back in, with a few more material examples (to satisfy my need for uniformly organized shader balls).

Reply:
Alright, I've added the glass back with a few extra examples (please see the last gif in the overview).